seeding/user.seeding.js

```diff
@@ -68,7 +74,10 @@ export const seedUsers = async (limit, role, customValues = {}) => {
 
       const table = [];
       users.forEach((user) => {
-        table.push({ Email: user.email, Password: customValues.password || DEFAULT_PASSWORD });
+        table.push({
+          Email: user.email,
+          Password: userDefaults.password || DEFAULT_PASSWORD,
```

Comment:
I think this can be replaced with `password` which is defined above.

```suggestion
          Password: password,
```